barretenberg/cpp/src/barretenberg/stdlib_circuit_builders/mock_circuits.hpp

```diff
@@ -42,13 +51,21 @@ class MockCircuits {
      * @param builder
      * @param num_gates
      */
-    template <typename Builder> static void add_arithmetic_gates(Builder& builder, const size_t num_gates = 4)
+    template <typename Builder>
+    static void add_arithmetic_gates(Builder& builder, const size_t num_gates = 4, bool random = true)
     {
         // For good measure, include a gate with some public inputs
```

Comment:
Not something you've added, but since you're going to push again will you remove this comment? There is no PI here.